lib/newPage.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
It's overkill, but it's fine.  I can imagine us doing a more elaborate page here at some point, so this is good.